src/lib/datepicker/datepicker.ts

```diff
@@ -148,7 +148,8 @@ export class MdDatepicker<D> implements OnDestroy {
   /** Whether the datepicker pop-up should be disabled. */
   @Input()
   get disabled() {
-    return this._disabled === undefined ? this._datepickerInput.disabled : this._disabled;
+    return this._disabled === undefined && this._datepickerInput ?
+        this._datepickerInput.disabled : this._disabled;
```

Comment:
Could you write a unit test that captures the situation in which `this._datepickerInput` would be undefined?